minecraft_servers/matherion/manifest.json

```diff
@@ -6,11 +6,14 @@
     "%.matherion.eu"
   ],
   "social": {
-    "web": "https://matherion.eu/",
+    "web": "https://matherion.eu",
+    "web-shop": "https://matherion.craftingstore.net",
+    "web-support": "https://discord.com/invite/mXTBfuS",
```

Comment:
That is not the meaning behind it. [A URL to a website is to be specified here](https://github.com/LabyMod/server-media/blob/master/docs/Manifest.md). If there is no website for support, then please remove the line.
It is not a required key.